app/org/sagebionetworks/bridge/play/controllers/ParticipantController.java

```diff
@@ -176,7 +177,7 @@ public Result getRequestInfo(String userId) throws Exception {
     }
     
     public Result updateParticipant(String userId) {
-        UserSession session = getAuthenticatedSession(RESEARCHER);
+        UserSession session = getAuthenticatedSession(ADMIN, RESEARCHER);
         Study study = studyService.getStudy(session.getStudyIdentifier());
 
         StudyParticipant participant = parseJson(request(), StudyParticipant.class);
```

Comment:
At some point I may add ADMIN to getParticipants() so that in the UI, you can search for and retrieve the participant. Right now, I am usually also a researcher at the point I am setting up a study.